docs/package_directory_structure/README.md

```diff
@@ -0,0 +1,43 @@
+# Package directory structure
+Content code entities in Demisto are presented by YAML files.
+
+Automation scripts and integrations [YAML files](https://github.com/demisto/content/blob/master/docs/yaml-file-integration/README.MD) include Javascript/Python code.
+
+Python scripts and integrations are stored in directories as packages.
+
+This is required for running [linting](https://github.com/demisto/content/blob/master/docs/linting/README.md) and [unit testing](https://github.com/demisto/content/blob/master/docs/tests/unit-testing/README.md) on the Python code.
+
+The package is converted into a YAML file, which include all the package components, in Demisto's CI build, using the `package_creator.py` script, and then populated in the Content release.
+
+---
+
+A package should be a sub-directory of either Integrations or Scripts directory, and contain the following files:
+
+ - `<INTEGRATION-NAME>.py` - Integration / automation script Python code.
```

Comment:
can we provide something similar to the following docs
![image](https://user-images.githubusercontent.com/7270217/64638478-86572700-d40e-11e9-87d9-892d0aea4e5f.png)
https://code.visualstudio.com/api/get-started/extension-anatomy